desc/objectives/linear_objectives.py

```diff
@@ -39,6 +40,223 @@ def update_target(self, eq):
             self.jit()
 
 
+# TODO: make this work with above, but for multiple target args?
+class FixParameter(_Objective):
+    """Fix specific degrees of freedom associated with a given Optimizable object.
+
+    Parameters
+    ----------
+    things : Optimizable
+        Object whose degrees of freedom are being fixed.
+    params : str or list of str
+        Names of parameters to fix. Defaults to all parameters.
+    target : dict of {float, ndarray}, optional
+        Target value(s) of the objective. Only used if bounds is None.
+        Should have the same tree structure as thing.params. Defaults to things.params.
+    bounds : tuple of dict {float, ndarray}, optional
+        Lower and upper bounds on the objective. Overrides target.
+        Should have the same tree structure as thing.params.
```

Comment:
or if it is meant to be `thing.params`, which of the `things` is it referring to?